src/print.rs

```diff
@@ -86,6 +86,14 @@ impl<'a, 'b> BacktraceFmt<'a, 'b> {
         // Currently a no-op-- including this hook to allow for future additions.
         Ok(())
     }
+
+    /// Write a string to the backtrace output.
+    ///
+    /// This is only for those messages which is not part of any frame,
+    /// and this won't increment the `frame_index` like frame
+    pub fn print(&mut self, msg: &str) -> fmt::Result {
```

Comment:
```suggestion
    pub fn message(&mut self, msg: &str) -> fmt::Result {
```
I think this is a better name than "print", which clearly explains that this is for inserting messages between frames.